client/site-profiler/components/site-profiler.tsx

```diff
@@ -164,9 +167,39 @@ export default function SiteProfiler( props: Props ) {
 						<LayoutBlockSection>
 							<MetricsMenu
 								basicMetricsRef={ basicMetricsRef }
+								performanceMetricsRef={ performanceMetricsRef }
+								healthScoresRef={ healthScoresRef }
 								onCTAClick={ () => setIsGetReportFormOpen( true ) }
 							/>
 							<BasicMetrics ref={ basicMetricsRef } basicMetrics={ basicMetrics.basic } />
+							<MetricsSection
```

Comment:
What do you think about creating a component to encapsulate all the display logic of advanced metrics? I was thinking in something like `<AdvancedMetrics... ` that will accept an array of refs (or just all the refs as different props) and the response of calling the `/site-profiler/advance` endpoint. 

That way, all the logic to iterate the response and decide what titles to show will be handled in a separate component that can grow independently of this one.